extensions/ql-vscode/src/authentication.ts

```diff
@@ -13,41 +14,12 @@ const SCOPES = ["repo", "gist", "read:packages"];
 /**
  * Handles authentication to GitHub, using the VS Code [authentication API](https://code.visualstudio.com/api/references/vscode-api#authentication).
  */
-export class Credentials {
+export class VSCodeCredentials implements Credentials {
```

Comment:
This file should probably live in `common/vscode/credentials.ts` since it's now part of the app and doesn't need to be accessed itself.